tensorflow/core/platform/default/build_config.bzl

```diff
@@ -662,34 +662,10 @@ def tf_additional_core_deps():
         clean_dep("//tensorflow:android"): [],
         clean_dep("//tensorflow:ios"): [],
         clean_dep("//tensorflow:linux_s390x"): [],
-        clean_dep("//tensorflow:windows"): [
-            "//tensorflow/core/platform/cloud:gcs_file_system",
-        ],
```

Comment:
This should still stay on?

Reply:
Thanks @mihaimaruseac. This part was trying to fix an unrelated error in CI at that time. I think it might not been needed now so I have removed the change in updated PR.